tests/testsuite/build.rs

```diff
@@ -215,21 +231,21 @@ fn cargo_compile_with_unsupported_short_config_flag() {
         .build();
 
     p.cargo("build -c net.git-fetch-with-cli=true")
-        .with_stderr(
-            "\
-error: unexpected argument '-c' found
+        .with_stderr_data(str![[r#"
+[ERROR] unexpected argument '-c' found
 
   tip: a similar argument exists: '--config'
 
 Usage: cargo[EXE] build [OPTIONS]
 
 For more information, try '--help'.
-",
-        )
+
+"#]])
         .with_status(1)
         .run();
 }
 
+#[allow(deprecated)]
 #[cargo_test]
 fn cargo_compile_with_workspace_excluded() {
```

Comment:
Does this need the deprecated functions?

Reply:
Updated to snapbox